DEVNOTES.md

```diff
@@ -14,26 +14,28 @@
 ***
 
 
-## Building Ignite
-Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project root directory (you can disable the tests when building using `-x test`):
+## Quick Start
+Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project
+root directory:
 ```
 gradlew clean build
 ```
-Upon build completion, CLI tool can be found be under `modules/cli/build` directory. Use `ignite` on Linux and MacOS, or `ignite.exe` on Windows.
-***
+Apart from building the project itself this command performs a few additional actions, for instance, testing. The build runs faster if
+these actions are disabled as described in the next section.
 
-# Gradle build
-
-## Prerequisites
-* Java 11 SDK
+To start an ignite-3 instance, package Apache Ignite 3 as described below and then follow [the user guide](https://ignite.apache.org/docs/3.0.0-beta/quick-start/getting-started-guide).
 ***
 
 
 ## Building Ignite
-Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project 
-root directory (you can disable the tests when building using `-x test`):
+Ignite follows standard guidelines for multi-module Gradle projects, so it can be easily built using the following command from the project
```

Comment:
Apache Ignite 3 follows standard guidelines for multi-module Gradle projects, so it can be built by using the following command from the project: